db/db.pl

```diff
@@ -2067,6 +2070,14 @@ sub parseArgs {
         $PREFIX .= "_" if ($PREFIX !~ /_$/);
     } elsif ($ARGV[0] =~ /-n$/) {
         $NOCHANGES = 1;
+    } elsif ($ARGV[0] =~ /--insecure$/) {
+        $SECURE = 0;
```

Comment:
You need to define all these vars up top, the tests are failing

https://cd.screwdriver.cd/pipelines/354/builds/132281/steps/test-viewer

Reply:
ugh... these merge errors are killing me